indexing-hadoop/src/test/resources/druid.test.data.tsv

```diff
@@ -0,0 +1,82 @@
+2011-04-10T00:00:00.000Z	spot	automotive	preferred	apreferred	113.221448
```

Comment:
We actually already went back and forth on this ;).  The thing is, the files are different.  While it might seem like redundant C&P, wouldn't changing the old file mean that we also have to change *all* tests that depend on the old file?  Those tests are working fine though and testing for various, meaningful things, I believe...

That is, if we just think of the new file as a new, different file (that happens to have a lot of stuff that looks the same as the old one) rather than a copy, is it so wrong to have different files that are capable of exercising different things?